db/package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "@nina-protocol/nina-db",
-  "version": "0.0.87",
+<<<<<<< HEAD
+  "version": "0.0.89",
+=======
+  "version": "0.0.84",
+>>>>>>> e12681bd8a3b79dae6d0661e16c6afb4b23e0a34
```

Comment:
Looks like we need to remove the diff merge conflict text here, guessing we want .89.